rag/svr/task_executor_refactor/dataset_structure_merger.py

```
@@ -14,143 +14,662 @@
+                {"kb_id": [kb_id], "knowledge_graph_kwd": [_DELETION_META_KWD]},
                 [],
                 OrderByExpr(),
                 offset,
-                page_size,
+                _PAGE_SIZE,
                 index,
                 [kb_id],
             )
-            field_map = settings.docStoreConn.get_fields(res, select_fields) or {}
+            fm = settings.docStoreConn.get_fields(res, ["id", "doc_id"]) or {}
         except Exception:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)